INSTALL.rst

```diff
@@ -269,11 +269,22 @@ Authentication and Authorization
 --------------------------------
 
 HaaS includes a pluggable architecture for authentication and authorization.
-HaaS ships with two authentication backends. One uses HTTP basic auth, with
-usernames and passwords stored in the haas database. The other is a "null"
-backend, which does no authentication or authorization checks. This can be
-useful for testing and experimentation but *should not* be used in production.
-You must enable exactly one auth backend.
+HaaS ships with a handful of authentication backends; you must enable exactly
+one of them. The list of incuded backends is:
```

Comment:
Could you:
a) list these in the same order they appear below?
b) append the module name (like `haas.ext.auth.database`) to each line?